tests/WE2E/end_to_end_tests.sh

```diff
@@ -70,18 +82,17 @@ sed -i 's|EXECDIR="${SR_WX_APP_TOP_DIR}/bin"|EXECDIR="${SR_WX_APP_TOP_DIR}/bin_i
 #-----------------------------------------------------------------------
 
 # Load Python Modules
-source ${APP_DIR}/env/wflow_${machine}.env
 cmd="${APP_DIR}/env/wflow_${machine}.env"
+source ${cmd}
```

Comment:
Reduced a bit of duplicate information here by using the variable cmd.

Reply:
Thanks for cleaning this up as well.